src/UploadHandler.php

```diff
@@ -138,7 +103,7 @@ public function handleUpload(UploadedFile $file) {
     }
 
     // Open temp file.
-    $tmp = "{$this->temporaryUploadLocation}/{$this->getFilename($file)}";
+    $tmp = 'temporary://' . $this->getFilename($file);
```

Comment:
I'd still make this configurable (but I agree with stream wrapper approach obviously).